docs/api/README.md

```diff
@@ -0,0 +1,302 @@
+## API Overview
+
+## Available endpoints
+Major resource endpoints supported by the Arrebol API are:
+
+| resource      | description                       |
+|:--------------|:----------------------------------|
+| `/queues`    | manages queue creation and status
+| `/jobs`      | manages job execution and tracking
+| `/workers`   | manages the addition of workers |
+
+### 1 - Queues
+
+#### 1.1 - Create a new queue
+
+**URL**
+```http
+POST /queues
+```
+
+**Request example**
+
+*Body*
+```json
+{
+   "name":"some_awesome_name",
+   "worker_nodes": [
+      {
+      	"address": "200.100.050.0",
+	 	"worker_pool": 5
+      },
+	  {
+       	"address": "200.100.050.1",
+		"worker_pool": 10
+      }
+   ],   
+}
+```
+
+**Response example**
+```json
+{
+	"id": "some_unique_id"
+}
+```
+
+#### 1.2 - Retrieves a list with the current queues
+
+**URL**
+
+```http
+GET /queues
+```
+
+**Response example**
+```json
+[
+    {
+      "id": "some_unique_id",
+      "name": "awesome_name",
+      "waiting_jobs": 2,
+      "worker_nodes": 5,
```

Comment:
replace worker_nodes by worker_pools?

Reply:
pools_size instead of worker_pool

Reply:
replace in other places too

Reply:
@emanueljoivo add the possible STATES in the README

Reply:
New version 1f3a9d353cca5e96b9bca4bfc39b0e7fda66ebd9.